packages/schemas/src/types/webauthn.ts

```diff
@@ -0,0 +1,50 @@
+import { z } from 'zod';
+
+export const webAuthnCreationOptionsGuard = z.object({
```

Comment:
```suggestion
export const webAuthnCreationOptionsGuard = z.object({
```
How about `webAuthnRegisterationOptionsGuard`?
To align with the `generateRegistrationOptions` API from the `simplewebauthn`.